plugin.video.esa/resources/lib/esa_addon.py

```diff
@@ -0,0 +1,245 @@
+# -*- coding: utf-8 -*-
+
+'''
```

Comment:
Also you probably want to look into spdx license headers to remove this big wall of text from your py files

Reply:
Hi enen92,
Thanks a lot for the advice about the spdx. I've removed the "big wall" along with coding line and replaced by the spdx headers.